config/locales/hi/restroom.hi.yml

```diff
@@ -0,0 +1,60 @@
+hi:
+  activerecord:
+    attributes:
+      restroom:
+        # only overridden attributes here, defaults work as expected
+        # (e.g. database field 'city' will be appear as 'City')
+        name: 'स्थान / जगह  / व्यवसाय का नाम'
+        street: 'सड़क का पता'
+        state: 'राज्य / प्रांत'
+        access: 'सुलभ?'
+        bath_type: 'रेस्टरूम का प्रकार'
+
+  search_bar:
+    enter_location: 'पता दर्ज करें'
+
+  restroom:
+    add_new: 'हमारे डेटाबेस में एक रेस्टरूम सबमिट करें'
+    preview: 'पूर्वावलोकन'
+    required: '* अंक आवश्यक फ़ील्ड'
+    guess_location: 'वर्तमान स्थान का अनुमान'
+    directions: 'दिशानिर्देश'
+    directions_hint: 'उदाहरण: पीछे की तीसरी मंजिल, ड्रेसिंग रूम द्वारा ...)'
+    comments_hint: '(उदाहरण: आपको "भुगतान करने वाला ग्राहक" होना है, बस कार्य करें जैसे आप थोड़ी देर के लिए ब्राउज़ कर रहे हैं ...)'
+    comments: 'टिप्पणियां'
+    accessible: 'सुलभ'
+    not_accessible: 'असुलभ '
+    changing_table: 'चेंजिंग टेबल '
+    no_changing_table: 'चेंजिंग टेबल नहीं'
+    type:
+      unisex: 'यूनिसेक्स / लिंग तटस्थ'
+      single_stall: 'जेंडर सिंगल स्टॉल या सुरक्षित जगह'
+    restsubmit: 'रेस्टरूम सहेजें'
+    rating:
+      positive: '%{percentage}% सकारात्मक'
+      unrated: 'अभी तक मूल्यांकित नहीं'
+    upvote: 'अच्छा हैं !'
+    downvote: 'सुरक्षित नहीं'
+    confirm:
+      upvote: 'क्या आप वाकई इस रेस्टरूम को अच्छी रेटिंग देना चाहते हैं?'
+      downvote: 'क्या आप वाकई इस रेस्टरूम को ख़राब रेटिंग देना चाहते हैं?'
+    flash:
+      field: 'एक आवश्यक फ़ील्ड खाली छोड़ दिया गया है। कृपया अपना सबमिशन पुनः जांचें और फिर फॉर्म सबमिट करें।'
+      unexpected: 'हम एक अप्रत्याशित समस्या में आ  गए, अगर समस्या जारी रहती है तो कृपया हमसे संपर्क करें।
```

Comment:
This line is missing a closing quotation mark `'`.

Everything else looks good!

Reply:
This was the error message, if anyone is particularly interested:

> I18n::InvalidLocaleData: can not load translations from /refugerestrooms/config/locales/hi/restroom.hi.yml: #<Psych::SyntaxError: (/refugerestrooms/config/locales/hi/restroom.hi.yml): did not find expected key while parsing a block mapping at line 42 column 7>

https://travis-ci.org/RefugeRestrooms/refugerestrooms/builds/436778300#L1888